### PR TITLE
x/batch: bump to 15 max messages

### LINKED ISSUE #926: Increase number of allowed TX in batch
**Is your feature request related to a problem? Please describe.**
From the reward account we want to send cash TX to the recipients via batch operation managed by gov. Unfortunately the limit is set fix to [10](https://github.com/iov-one/weave/blob/master/x/batch/decorator.go#L16).

**Describe the solution you'd like**
We need to support 15 (@kmw101). 

**Describe alternatives you've considered**

**Additional context**
From slack: A distribution contract is not "flexible" enough for the distribution.

**Acceptance criteria**
- [ ] 15 cash.Send TX work in a single batch

### PR DESCRIPTION
**Pull request by ruseinov**
Fixed #926

### CODE REVIEW

**Comment by ruseinov** [comment]
@alpe I take that this coverage issue is coming from gitian build merged? 
It seems pretty consistent across different version builds, curious what to do.

**Comment by alpe** [comment]
No, the issue not related to gitian. The gitian steps are later in the process and only for main go version. Feel free to rebase on top of master as I have removed the gitian steps already for a proper PR.

File: x/batch/decorator.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/tendermint/tendermint/libs/common"
 )
 
-const MaxBatchMessages = 10
+const MaxBatchMessages = 15
```
Comment (alpe) [inline x/batch/decorator.go]: q: no unit test failing... 🤔

**Comment by ruseinov** [comment]
Not sure what to do with failing test, but does not seem relevant to our case.

**Comment by ruseinov** [comment]
Figured it out, there were some tests depending on batch,,

File: x/batch/decorator.go (hunk 1 above)
Comment (ruseinov) [reply x/batch/decorator.go]: yeah, cause it depends on this value!

**Pull request merged**